src/lib/input/placeholder.html

```diff
@@ -0,0 +1,12 @@
+<label class="md-placeholder"
```

Comment:
It might be a problem if the placeholder is _always_ the label for the input. I can see cases where you would want to provide a more specific label than what's in the placeholder.